src/displayapp/screens/FirmwareUpdate.cpp

```diff
@@ -78,22 +78,20 @@ void FirmwareUpdate::Refresh() {
 }
 
 void FirmwareUpdate::DisplayProgression() const {
-  float current = bleController.FirmwareUpdateCurrentBytes() / 1024.0f;
-  float total = bleController.FirmwareUpdateTotalBytes() / 1024.0f;
-  int16_t pc = (current / total) * 100.0f;
-  sprintf(percentStr, "%d %%", pc);
-  lv_label_set_text(percentLabel, percentStr);
+  const uint32_t current = bleController.FirmwareUpdateCurrentBytes();
```

Comment:
Is this const correct?

Reply:
Since it's a local variable, it's initialized each time the function is called. I believe it's correct, because the variable isn't changed after initialization.